next-docs/pages/world/balance.mdx

```diff
@@ -1 +1,15 @@
 # Balance
+
+ETH is stored in the `World` contract and owned by namespaces.
+Any ETH sent with a call to a `System` is kept on the `World`, and credited to the namescape to which the `System` belongs.
+This has two important effects:
+
+- The ETH balance is shared between the different `System` contracts in the same namespace of the same `World`.
+- The same `System` can be used in multiple `World` contracts, and in multiple namespaces in the same `World`, and the ETH balances remain separate.
+
+To know how much ETH a call has included a `System` can check [`_msgValue()`](https://github.com/latticexyz/mud/blob/main/packages/world/src/WorldContext.sol#L34-L36).
+
+To know how much ETH a specific namespace has you can look in `world:Balances` (the `Balances` table of the `world` namespace).
```

Comment:
This might be a good opportunity to include a small code snippet for illustration!

```solidity
import { Balances } from "@latticexyz/world/src/codegen/tables/Balances.sol";

uint256 balance = Balances.get(<namespace>);

```

Reply:
I can't do that. I don't include code snippets unless I manage to run them and they work first, and trying to read the information causes a revert. See https://discord.com/channels/865335009915961364/1156299216575012946/1167457348613308457

Reply:
did you manage to run the script after explicitly passing the rpc url ?